package-test.js

```diff
@@ -33,6 +33,7 @@ const looper = () => {
       const latestVersion = info['dist-tags'][CUSTOM_DIST[_name] || 'latest'];
       const latestVersionTime = info['time'][latestVersion];
       if (
+        _name !== 'webpack' &&
```

Comment:
can't we just add webpack to the EXCEPTIONS array?

Reply:
yea this!